### ISSUE TITLE
pdf 모델링

### ISSUE THREAD

**Issue by Kimyungi** [jayleenym/AYOA#8]
- [x] input 정의: text, mask
- [x] input에 맞는 label 정의: pdf 날짜에 맞는 상승/하락 정보
- [x] Model 구조 정의: Attention 기반
- [x] Dataloader 정의 및 loss 정의
- [x] 학습 및 training loss, training acc 확인
- [x] dataset을 train / val / test로 나누어 정확도 확인
